src/inmanta/data/__init__.py

```diff
@@ -1713,6 +1713,7 @@ async def query_resource_actions(
         attribute_value: Optional[str] = None,
         log_severity: Optional[str] = None,
         limit: int = 0,
+        offset: Optional[int] = None,
```

Comment:
The ticket mentions in the _"Notes on Paging"_ section that offset-based pagination should not be used, because databases don't have an efficient way to execute these types of queries.